pages/linux/mangohud.md

```diff
@@ -6,3 +6,15 @@
 - Use `mangohud` on top of an application:
 
 `mangohud {{command}}`
+
+- Change overlay position:
+
+`<RightShift F11>`
+
+- Change overlay verbosity:
+
+`<RightShift F10>`
+
+- Toggle overlay visibility:
+
+`<RightShift F12>`
```

Comment:
```suggestion
`<ShiftRight F11>`

- Change overlay verbosity:

`<ShiftRight F10>`

- Toggle overlay visibility:

`<ShiftRight F12>`
```
I'm asking for opinions. Which one looks better? RightShift sounds more natural and ShiftRight would be uniform with ArrowRight